CONTRIBUTING.md

```diff
@@ -1,72 +1,46 @@
+project! Patches and other forms of community development help make the
+AtoM project vibrant and responsive to users' needs. Here are the guidelines
+on reporting issues and making pull requests to the AtoM project.
 
-Thanks!
 
 **Contents**
 
+* [Reporting a security vulnerabilitiy](#security-vulnerability)
 * [Reporting general bugs](#reporting-general-bugs)
 * [Contributing code](#contributing-code)
 * [Contributing documentation and translations](#contributing-documentation-and-translations)
 
-## Reporting general bugs
-
-**If you are reporting a security vulnerability**, please consult the
-instructions in our SECURITY.md file before proceeding. If you have discovered
-an issue in AtoM that is **not** related to a security vulnerability, we
-welcome your reports.
-
-You can report general bugs in two ways: 
-
-* By creating a new post in the [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
-* By [opening an issue](https://github.com/artefactual/atom/issues) in our
-  GitHub repository
-
-Please be sure to include all the information we will need to reproduce the
-issue locally. This includes:
+## Security vulnerability
 
-* The version of AtoM you are using
-* Basic information about your installation environment, including PHP, MySQL,
-  Elasticsearch, and operating system versions
-* Steps to reproduce the issue
-* The resulting error or vulnerability and the expected outcome
-* If there are any error logs related to the issue, please include the
-  relevant parts as well
+**If you are reporting a security vulnerability**, refer yourself to the
+[SECURITY.md](./SECURITY.md) document. 
 
-You can find useful tips on how to find this information in the Troubleshooting 
-page in our documentation:
+## Reporting general bugs
 
-* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/
+If you have discovered an issue in AtoM, we welcome your reports.
 
-Note as well that, as a community-driven open source project, we depend on our
-community to be able to maintain and develop AtoM. We are committed to
-including as many bug fixes as we can in each new release, but a confirmed
-report is not enough alone to guarantee that a fix will be included in the
-next release. If your institution is interested in sponsoring a fix, feel free
-to contact Artefactual Systems for an estimate - all sponsored development
-will be included in the next public release. For more information on how we
-develop and maintain AtoM, please see:
+Start by doing a search on [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
+and in the [AtoM GitHub issues](https://github.com/artefactual/atom/issues)
+to see if your question has been considered in the past.
 
-* https://wiki.accesstomemory.org/Development/Philosophy
+If nothing comes up, start a conversation by opening a new thread on the user
+forum. When your bug will have been better understood, you may be invited to
```

Comment:
The tense is a bit strange in the second sentence here. I'd say something more like: 

> "If your issue is confirmed to be a bug in the application, you may be..."

or

> "If we manage to reproduce your issue, you may be..."

or 

> "Once we understand the issue better, you may be..."

However, I've gotta point out that even with your modifications (or mine), this still kind of implies that Artefactual folks will be active in the forum and helping to determine if something is a bug or not. 

Which.... maybe isn't the case?

(similarly, earlier we encourage folks to search the forum first. but we're not maintaining the tags that help make searching easier? just a thought...)

Just be sure you are considering it from an excited new contributor's perspective. If the new volunteer Ambassadors etc will be more regularly active in the forum, or you or the Maintainers or someone else will be starting to check more regularly, etc, it's fine. But if not, I think directing people to do something that we won't follow up may just lose us potential contributors, and in that case I would skip the posting in the forum part, and just tell them to file an issue. Maintainers may have to sort through more false reports that way, but not look at the forum at least, or imply they will be looking.

Reply:
Another way to say this could be:

```
If you are sure that the issue you have is a bug, please open an issue on GitHub. If not, feel welcome to open a thread in the Users Group forum.
```

As far as I understand, we are trying to keep the issues tidy and focused on bugs, and to have more general discussions or issues about using the software as it is in the User Group forum. I just want an actionable way to say this ?

Reply:
That sounds like a good update to me!